api/Plugin.json

```diff
@@ -0,0 +1,529 @@
+{
+  "api": {
+    "Plugin": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Plugin",
+        "support": {
+          "webview_android": [
+            {
+              "version_added": true,
```

Comment:
The `"version_added": "59"` data should be at the top for each of these. Whatever is the latest info should be the top-most item. I think this is true for all the features in this file.

Reply:
Oh right. I'm on it..